### PR TITLE
docs: Add HOME and fix link

### PR DESCRIPTION
**Pull request by NHingerl**
## Description

Changes proposed in this pull request:

- add HOME to sidebar
- fix link to blog post

## Related Issues and Documents

Closes:

Related issues:

https://github.com/kyma-project/telemetry-manager/issues/322

### CODE REVIEW

File: docs/user/_sidebar.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-- [Kyma Telemetry Module](README.md)
-- [Telemetry Manager](01-manager.md)
-- [Application Logs](02-logs.md)
-- [Traces](03-traces.md)
-- [Metrics](04-metrics.md)
-- [Tutorials](05-tutorials.md)
-- [Resources](resources/README.md)
\ No newline at end of file
+- [Home](https://kyma-project.io/#/)
```
Comment (a-thaler) [inline docs/user/_sidebar.md:1]: Seeing that one it does not have the intendation as you propose: https://raw.githubusercontent.com/kyma-project/serverless-manager/main/docs/user/_sidebar.md

Comment (IwonaLanger) [reply docs/user/_sidebar.md:1]: I also think we should use [Home] (/_) _(space added on purpose)_. Unless there's been a change I'm not aware of.

**Review by a-thaler — APPROVED** [review]

File: docs/user/_sidebar.md (hunk 1 above)
Comment (NHingerl) [reply docs/user/_sidebar.md]: I removed the indentation and changed the link to a relative one. Thank you both for spotting this!

**Review by IwonaLanger — APPROVED** [review]

**Comment by NHingerl** [comment]
Arrgh no, the link in the resource file should have been adjusted in the `.go` file and than populated with `make manifests`, right?

**Review by chrkl — DISMISSED** [review]

File: config/crd/bases/telemetry.kyma-project.io_logpipelines.yaml [hunk 2]
```diff
@@ -137,7 +137,7 @@ spec:
                     description: Configures an output to the Kyma-internal Loki instance.
                       [Fluent Bit grafana-loki output](https://grafana.com/docs/loki/v2.2.x/clients/fluentbit/).
                       **Note:** This output is considered legacy and is only provided
-                      for backward compatibility with the [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/)
+                      for backward compatibility with the [deprecated](https://github.com/kyma-project/website/blob/main/content/blog-posts/2023-01-17-release-notes-2.10/index.md#deprecation-of-monitoring)
```
Comment (IwonaLanger) [inline config/crd/bases/telemetry.kyma-project.io_logpipelines.yaml:140]: Shouldn't the outdated link be replaced with [this one](https://github.com/kyma-project/website/blob/main/content/blog-posts/2022-11-02-loki-deprecation/index.md)?

File: docs/user/resources/02-logpipeline.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ For details, see the [LogPipeline specification file](https://github.com/kyma-pr
...
+| **output.&#x200b;grafana-loki**  | object | Configures an output to the Kyma-internal Loki instance. [Fluent Bit grafana-loki output](https://grafana.com/docs/loki/v2.2.x/clients/fluentbit/). **Note:** This output is considered legacy and is only provided for backward compatibility with the [deprecated](https://github.com/kyma-project/website/blob/main/content/blog-posts/2023-01-17-release-notes-2.10/index.md#deprecation-of-monitoring) in-cluster Loki instance. It might not be compatible with the latest Loki versions. For integration with a custom Loki installation use the `custom` output with the name `loki` instead, see also [Installing a custom Loki stack in Kyma](https://github.com/kyma-project/examples/tree/main/loki). |
```
Comment (IwonaLanger) [inline docs/user/resources/02-logpipeline.md:93]: Again, shouldn't the outdated link be replaced with [this one](https://github.com/kyma-project/website/blob/main/content/blog-posts/2022-11-02-loki-deprecation/index.md)?

File: config/crd/bases/telemetry.kyma-project.io_logpipelines.yaml (hunk 2 above)
Comment (NHingerl) [reply config/crd/bases/telemetry.kyma-project.io_logpipelines.yaml:140]: It probably should but I'm not going to touch the go or yaml file again :D

File: docs/user/resources/02-logpipeline.md (hunk 3 above)
Comment (NHingerl) [reply docs/user/resources/02-logpipeline.md:93]: It probably should but I'm not going to touch the go or yaml file again :D

**Review by IwonaLanger — APPROVED** [review]

**Review by chrkl — APPROVED** [review]

**Pull request merged**